htdocs/changesshkey.php

```diff
@@ -40,8 +40,10 @@
 }
 if (isset($_POST["password"]) and $_POST["password"]) { $password = strval($_POST["password"]); }
  else { $result = "passwordrequired"; }
-if (isset($_POST["sshkey"]) and $_POST["sshkey"]) { $sshkey = strval($_POST["sshkey"]); }
- else { $result = "sshkeyrequired"; }
+if (isset($_POST["sshkey"]) and $_POST["sshkey"]) {
+    $sshkey = strval($_POST["sshkey"]);
+    if (! check_sshkey($sshkey)) { $result = "invalidsshkey"; }
```

Comment:
I don't feel confident about having a mandatory validation if the validation is not unit tested with a range of public keys.

Reply:
CI would run tests for dsa, ecdsa, rsa, ed22519.
Let me know if you have others in mind.